sonarqube/assets/configuration/spec.yaml

```diff
@@ -73,32 +90,14 @@ files:
                   - security.
                 exclude:
                   - issues.(false_positive_issues|reopened_issues)
-
-          Note: This is required for all project checks, whether `is_jmx` is set to `true`
-          or `false.
-        options:
-          - name: tag
-            description: |
-              Custom tag for the specified component.
-            value:
-              type: string
-          - name: include
-            description: |
-              List of metrics to include for the specified component.
-            value:
-              type: array
-              items:
-                type: string
-          - name: exclude
-            description: |
-              List of metrics to exclude for the specified component.
-            value:
-              type: array
-              items:
-                type: string
+        value:
+          type: object
+          additionalProperties: true
```

Comment:
What is the exact purpose of additionalProperties?

Reply:
discussed over zoom, tl;dr `additionalProperties` allows configuration of additional keys that aren’t defined in the config spec